### PR TITLE
update codeowners

### PR DESCRIPTION
**Pull request by bruno-garcia**
Adding @SeanFeldman and @jamescrosswell 

#skip-changelog

### CODE REVIEW

**Review by bitsandfoxes — APPROVED** [review]
🔥

**Comment by bruno-garcia** [comment]
We agreed that for large impact PRs, the author will wait until more approvers come in. But for the sake of speed, we'll let any single approver unblock the PR

**Pull request merged**
